.github/workflows/node-tests.yml

```diff
@@ -4,25 +4,10 @@ on:
   pull_request:
 
 jobs:
-  npm-ci:
-    name: (npm) Install, build, and test
-    runs-on: ubuntu-latest
-    steps:
-      - name: Cancel Previous Runs
-        uses: styfle/cancel-workflow-action@0.11.0
-        with:
-          all_but_latest: true
-      - name: Checkout code
-        uses: actions/checkout@v3
-      - name: Setup Node
-        uses: actions/setup-node@v3
-        with:
-          node-version: 16
-      - name: Install node dependencies (npm ci)
-        run: npm ci
-      - name: Run npm lint
-        run: npm run lint
-      - name: Run npm test
-        run: npm run test
-      - name: Run npm build
-        run: npm run build
+  node-tests:
+    uses: alleyinteractive/.github/.github/workflows/node-tests.yml@main
+    with:
+      ci: true
+      node: 16
+      run-audit: false
```

Comment:
Those are already the default values.
```suggestion
```

Reply:
However, the root workflow could be updated to have different defaults, which could break this workflow. I'm fine with having them be explicitly set here, even if it duplicates the current defaults.

Reply:
Good point!